includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -1825,26 +1822,6 @@ public static function payment_gateways_rewewal_support( $gateway ) {
 		return self::payment_gateways_renewal_support( $gateway );
 	}
 
-	/**
-	 * Do not display formatted order total on the Edit Order administration screen
-	 *
-	 * @since 1.0.0 - Migrated from WooCommerce Subscriptions v1.5.17
-	 */
-	public static function maybe_remove_formatted_order_total_filter( $formatted_total, $order ) {
```

Comment:
Even though this function isn't doing anything since 2.0 (2015), and I wouldn't expect anyone would be calling it, if someone was calling it via `WC_Subscriptions_Admin::maybe_remove_formatted_order_total_filter()` their site would fatal after updating. 

For that reason, in WC Subscriptions we never straight up remove a function like this. We should technically deprecate it.

Reply:
Oh, gotcha! I have deprecated it instead 👍